src/backend/joanie/core/serializers.py

```diff
@@ -364,15 +319,28 @@ class EnrollmentSerializer(serializers.ModelSerializer):
     """
 
     id = serializers.CharField(source="uid", read_only=True, required=False)
-    user = serializers.CharField(source="user.username", read_only=True, required=False)
-    course_run = serializers.SlugRelatedField(
-        queryset=models.CourseRun.objects.all(), slug_field="resource_link"
-    )
+    course_run = CourseRunSerializer(read_only=True)
 
     class Meta:
         model = models.Enrollment
-        fields = ["id", "user", "course_run", "is_active", "state"]
-        read_only_fields = ["id", "user", "state"]
+        fields = ["id", "course_run", "is_active", "state"]
```

Comment:
Why not keep the user field as a reference?

Reply:
As this ressource is requested by an authenticated user, I wonder if it was necessary to embed user information within this ressource.